code-samples/creating-your-bot/configuration-files/index.js

```diff
@@ -7,7 +7,8 @@ client.once('ready', () => {
 });
 
 client.on('message', message => {
-	if (message.content === '!ping') {
+	if (!message.content.startsWith(config.prefix)) return;
```

Comment:
Looked at the guide flow again, these resulting code snippets should reflect what's taught in the guide and this is at the very beginning, where we just introduced directly comparing the message content, rather than dissecting it and returning if it doesn't start with the prefix. Line 10 is not mentioned in the respective guide https://discordjs.guide/creating-your-bot/#replying-to-messages yet (previous state of the guide before entering this section)

Reply:
@almostSouji that section of code is also referenced and linked here

https://discordjs.guide/creating-your-bot/configuration-files.html#implementing-your-config-file

and this is what I was working from. It feels weird for the guide to mention putting things within configuration and then not use said configuration.

Reply:
I'm referring to line 2. not how 3 is changed to use the configured prefix, that's absolutely valid.

Reply:
Yeah - you're right. I think I came at that from the approach of "I would do this within my application" but it's out of scope for the tutorials that consume it.

To confirm, drop line 2?